src/pint/models/stand_alone_psr_binaries/DD_model.py

```diff
@@ -415,17 +414,34 @@ def d_beta_d_DTH(self):
         return self.a1() / c.c * (-eTheta) / np.sqrt(1 - eTheta**2) * cosOmg
 
     ##################################################
-    def Dre(self):
-        """Dre defined in T. Damour and N. Deruelle(1986)equation [48]
+
+    def delayR(self):
+        """Roemer delay defined in T. Damour and N. Deruelle (1986)
 
         Computes::
 
-            Dre = alpha*(cos(E)-er)+(beta+gamma)*sin(E)
+            delayR = alpha*(cos(E)-er) + beta*sin(E)
         """
         er = self.er()
         sinE = np.sin(self.E())
         cosE = np.cos(self.E())
-        return self.alpha() * (cosE - er) + (self.beta() + self.GAMMA) * sinE
+        return self.alpha() * (cosE - er) + self.beta() * sinE
+
+    ##################################################
+    def Dre(self):
+        """Dre defined in T. Damour and N. Deruelle (1986) equation [48]
+
+        Computes::
+
+            Dre = alpha*(cos(E)-er)+(beta+gamma)*sin(E)
+        """
+
+        # er = self.er()
```

Comment:
Any need to keep these commented lines?

Reply:
The docstring could also help understand that the equation is equivalent to the expression in the code by giving what delayR and delayE are here.  otherwise it looks like an error.

Reply:
I have modified this.